### PR TITLE
Lip Fix

### PR DESCRIPTION
**Pull request by bubbliiiing**
EN：
1. Prompt Modification: The prompt has been updated for better clarity and user experience.
2. Reconstruction of the Second Area: Changes have been made in the second area, including the addition of 'eyes' as a new feature.
3. Color Transfer Area Adjustment: Modifications have been applied to the color transfer area for optimization.

CN：
1. 提示符修改：为了提高清晰度和用户体验，提示符已经进行了更新。
2. 第二区域已经进行了修改，新增了“眼睛”作为一个新的特性。
3. 颜色转换区域调整：对颜色转换区域进行了优化性的修改，在颜色空间转换和分布统计中，修正原有的cv2.COLOR_BGR2LAB 为 cv2.COLOR_RGB2LAB。

### CODE REVIEW

File: scripts/easyphoto_infer.py [hunk 1]
```diff
@@ -529,13 +533,13 @@ def easyphoto_infer_forward(
                     logging.info("Start color shift last.")
                     # scale box
                     rescale_retinaface_box = [int(i * default_hr_scale) for i in input_image_retinaface_box]
+                    second_diffusion_output_image = np.array(second_diffusion_output_image)
+                    face_skin_mask = np.int32(np.float32(face_skin(Image.fromarray(np.uint8(second_diffusion_output_image[rescale_retinaface_box[1]:rescale_retinaface_box[3], rescale_retinaface_box[0]:rescale_retinaface_box[2],:])), retinaface_detection, needs_index=[[1, 2, 3, 4, 5, 10]])[0]) > 128)
```
Comment (wuziheng) [inline scripts/easyphoto_infer.py:537]: 说实话，不算刁难~ 这个代码的可读性太差了，把 (np.uint8(second_diffusion_output_image[rescale_retinaface_box[1]:rescale_retinaface_box[3], rescale_retinaface_box[0]:rescale_retinaface_box[2],:] 这个变量加一行拿出来然后命名引导一下会好一点。不改也没事。。。。

File: scripts/easyphoto_infer.py [hunk 2]
```diff
@@ -80,13 +82,15 @@ def get_controlnet_unit(unit, input_image, weight):
...
                                             guidance_end=1,
+                                            control_mode=1,
                                             resize_mode='Just Resize',
                                             model='control_sd15_random_color')
     elif unit == "tile":
         control_unit = ControlNetUnit(
             input_image=input_image, module='tile_resample',
             weight=weight,
             guidance_end=1,
+            control_mode=1,
```
Comment (wuziheng) [inline scripts/easyphoto_infer.py:93]: 可以加上一个注释，方便不熟悉webui controlnet 接口的人理解，=1 的改进是 prompt is more important

File: scripts/easyphoto_infer.py [hunk 3]
```diff
@@ -485,18 +489,15 @@ def easyphoto_infer_forward(
...
-                    
                     first_diffusion_output_image[input_image_retinaface_box[1]:input_image_retinaface_box[3], input_image_retinaface_box[0]:input_image_retinaface_box[2],:] = \
                         first_diffusion_output_image_face_area * face_skin_mask + first_diffusion_output_image[input_image_retinaface_box[1]:input_image_retinaface_box[3], input_image_retinaface_box[0]:input_image_retinaface_box[2],:] * (1 - face_skin_mask)
                     first_diffusion_output_image = Image.fromarray(first_diffusion_output_image)
-                    
-                # Obtain the mask of the area around the face
-                input_mask  = Image.fromarray(np.uint8(cv2.dilate(np.array(origin_input_mask), np.ones((96, 96), np.uint8), iterations=1) - cv2.erode(np.array(origin_input_mask), np.ones((48, 48), np.uint8), iterations=1)))
+                    first_diffusion_output_image.save("1.jpg")
```
Comment (wuziheng) [inline scripts/easyphoto_infer.py:500]: debug code should delete

Comment (bubbliiiing) [reply scripts/easyphoto_infer.py:500]: 👴 is sb.

File: scripts/easyphoto_infer.py (hunk 1 above)
Comment (bubbliiiing) [reply scripts/easyphoto_infer.py:537]: yes改一下

**Comment by wuziheng** [comment]
tested and refined by Wzh, improvements is obvious!
after
before

**Review by wuziheng — APPROVED** [review]
LGTM

**Pull request merged**